smoke/IotEdgeQuickstart/details/Details.cs

```diff
@@ -273,7 +277,7 @@ protected Task ConfigureBootstrapper()
                 agentImage = Option.Some<string>(this.EdgeAgentImage());
             }
 
-            return this.bootstrapper.Configure(method, agentImage, this.hostname, this.deviceCaCert, this.deviceCaPk, this.deviceCaCerts, this.runtimeLogLevel);
+            return this.bootstrapper.Configure(method, agentImage, this.hostname, this.parentHostname, this.deviceCaCert, this.deviceCaPk, this.deviceCaCerts, this.runtimeLogLevel);
```

Comment:
Did you forget to update `Configure(8)` in `iotedge\smoke\IotEdgeQuickstart\details\IotedgedWindows.cs`? It takes the argument but it doesn't do the `parentHostname.ForEach(v => doc.ReplaceOrAdd("parent_hostname", v));`

Reply:
I added the comment in IotedgedWindows.cs to say that Windows is no longer supported in 1.1.0 or later.  Therefore I didn't make any change to use it.